client/src/components/commonComponents/Tabs/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, Fragment } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Paper from '@material-ui/core/Paper';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
```

Comment:
replace all of that imports with one import from the '@material-ui/core'
and put all of the things you need inside { } 

`import { Toolbar, Step ... } from '@material-ui/core';`

Reply:
alright :+1: